watcher_latency.go

```diff
@@ -18,23 +18,30 @@ func (lw *LatencyWatcher) infoKeys() []string {
 }
 
 func (lw *LatencyWatcher) detailKeys(rawMetrics map[string]string) []string {
-	return []string{"latency:"}
+	return []string{"latencies:", "latency:"}
 }
 
 func (lw *LatencyWatcher) refresh(infoKeys []string, rawMetrics map[string]string, ch chan<- prometheus.Metric) error {
-	latencyStats := parseLatencyInfo(rawMetrics["latency:"])
+	latencyStats := make(map[string]StatsMap)
+
+	if rawMetrics["latencies:"] != "" {
+		latencyStats = parseLatencyInfo(rawMetrics["latencies:"])
+	} else {
+		latencyStats = parseLatencyInfoLegacy(rawMetrics["latency:"])
+	}
+
 	log.Tracef("latency-stats:%+v", latencyStats)
 
 	for namespaceName, nsLatencyStats := range latencyStats {
 		for operation, opLatencyStats := range nsLatencyStats {
 			for i, labelValue := range opLatencyStats.(StatsMap)["bucketLabels"].([]string) {
 				// aerospike_latencies_<operation>_bucket metric - Less than or equal to histogram buckets
-				pm := makeMetric("aerospike_latencies", operation+"_ms_bucket", mtGauge, config.AeroProm.MetricLabels, "cluster_name", "service", "ns", "le")
+				pm := makeMetric("aerospike_latencies", operation+"_"+opLatencyStats.(StatsMap)["timeUnit"].(string)+"_bucket", mtGauge, config.AeroProm.MetricLabels, "cluster_name", "service", "ns", "le")
```

Comment:
Is this backwards compatible? The server returns `msec` and the like, while the old system was strictly "ms". If not, how much of an issue is it?